Explorer/Assets/DCL/AvatarRendering/AvatarShape/Rendering/TextureArray/TextureArrayHandler.cs

```diff
@@ -127,18 +139,18 @@ internal Texture2DArray GetDefaultTextureArray(Vector2Int resolution)
             return GetOrCreateSlotHandler(resolution).arrays[DEFAULT_SLOT_INDEX];
```

Comment:
I think you need to change this `DEFAULT_SLOT_INDEX` value to the default slot index used, since its not always 0

Reply:
done